### PR TITLE
Chore: reuse metrics names in scheduler perf-test

### PR DESCRIPTION
**Pull request by nikzayn**
#### What type of PR is this?
/kind bug
/kind cleanup
/sig scheduling

#### What this PR does / why we need it:
This PR updates the naming convention for better understanding to make the context clear and we need to maintain the codebase.

#### Which issue(s) this PR fixes:
[116204](https://github.com/kubernetes/kubernetes/issues/116204)

#### Special notes for your reviewer:
Just re-check for the changes

#### Does this PR introduce a user-facing change?
None

### CODE REVIEW

File: pkg/scheduler/metrics/metrics.go [hunk 1]
```diff
@@ -28,6 +28,8 @@ import (
 const (
 	// SchedulerSubsystem - subsystem name used by scheduler.
 	SchedulerSubsystem = "scheduler"
+	// SchedulingAttemptDurationSeconds - To schedule the attempts for duration
+	SchedulingAttemptDurationSeconds = "scheduling_attempt_duration_seconds"
```
Comment (kerthcet) [inline pkg/scheduler/metrics/metrics.go:32]: This should be reused in L74.

File: pkg/scheduler/metrics/metrics.go [hunk 2]
```diff
@@ -28,6 +28,8 @@ import (
 const (
 	// SchedulerSubsystem - subsystem name used by scheduler.
 	SchedulerSubsystem = "scheduler"
+	// SchedulingAttemptDurationSeconds - To schedule the attempts for duration
```
Comment (kerthcet) [inline pkg/scheduler/metrics/metrics.go:31]:
```suggestion
	// SchedulingAttemptDurationSeconds - Scheduling attempt latency in seconds
```

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 3]
```diff
@@ -82,12 +83,15 @@ const (
 	extensionPointsLabelName = "extension_point"
 )
 
+var schedulerSubsystemAttemptDurationSeconds = metrics.SchedulerSubsystem + "_" + metrics.SchedulingAttemptDurationSeconds
```
Comment (kerthcet) [inline test/integration/scheduler_perf/scheduler_perf_test.go:86]: It should be const

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 4]
```diff
@@ -82,12 +83,15 @@ const (
...
+var metricLabelValues = []string{"Filter", "Score"}
```
Comment (kerthcet) [inline test/integration/scheduler_perf/scheduler_perf_test.go:87]: remove this, instead, use `metrics.Filter` and `metrics.Score` directly below.

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 5]
```diff
@@ -82,12 +83,15 @@ const (
...
 var (
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
 		Metrics: map[string]*labelValues{
-			"scheduler_framework_extension_point_duration_seconds": {
+			schedulerSubsystemAttemptDurationSeconds: {
 				label:  extensionPointsLabelName,
-				values: []string{"Filter", "Score"},
+				values: metricLabelValues,
 			},
 			"scheduler_scheduling_attempt_duration_seconds": nil,
```
Comment (kerthcet) [inline test/integration/scheduler_perf/scheduler_perf_test.go:96]: Same to these two metrics too.

**Comment by kerthcet** [comment]
/retitle Chore: reuse metrics names in scheduler perf-test

File: pkg/scheduler/metrics/metrics.go (hunk 1 above)
Comment (nikzayn) [reply pkg/scheduler/metrics/metrics.go:32]: Sure, I will do that too.

File: pkg/scheduler/metrics/metrics.go (hunk 2 above)
Comment (nikzayn) [reply pkg/scheduler/metrics/metrics.go:31]: Updating it.

File: test/integration/scheduler_perf/scheduler_perf_test.go (hunk 3 above)
Comment (nikzayn) [reply test/integration/scheduler_perf/scheduler_perf_test.go:86]: Sure thing

File: test/integration/scheduler_perf/scheduler_perf_test.go (hunk 4 above)
Comment (nikzayn) [reply test/integration/scheduler_perf/scheduler_perf_test.go]: I have fixed this change in my new commit

File: test/integration/scheduler_perf/scheduler_perf_test.go (hunk 5 above)
Comment (nikzayn) [reply test/integration/scheduler_perf/scheduler_perf_test.go]: Updated this too.

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 6]
```diff
@@ -78,19 +79,20 @@ const (
...
-	extensionPointsLabelName = "extension_point"
+	configFile                               = "config/performance-config.yaml"
+	extensionPointsLabelName                 = "extension_point"
+	schedulerSubsystemAttemptDurationSeconds = metrics.SchedulerSubsystem + "_" + metrics.SchedulingAttemptDurationSeconds
 )
 
 var (
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
 		Metrics: map[string]*labelValues{
-			"scheduler_framework_extension_point_duration_seconds": {
```
Comment (kerthcet) [inline test/integration/scheduler_perf/scheduler_perf_test.go:88]: FYI: `scheduler_framework_extension_point_duration_seconds`, `scheduler_scheduling_attempt_duration_seconds` and `scheduler_pod_scheduling_duration_seconds` are three different names.

**Comment by kerthcet** [comment]
/close

As we have more works to do refer to https://github.com/kubernetes/kubernetes/issues/116204#issuecomment-1563895137

**Pull request closed without merging**